rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/test_is_economizer_modeled_in_proposed.py

```diff
@@ -0,0 +1,208 @@
+from rct229.rulesets.ashrae9012019.ruleset_functions.baseline_systems.baseline_hvac_sub_functions.is_economizer_modeled_in_proposed import (
+    is_economizer_modeled_in_proposed,
+)
+from rct229.schema.validate import schema_validate_rmr
+
+TEST_RMD_FIXED_TYPE = {
```

Comment:
Should this be named `TEST_RMI_FIXED_TYPE`?

Reply:
Addressed.